bot.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
```

Comment:
first of all, I believe you could write here simply `# coding: utf-8` instead. Secondly, I believe this encoding line is no longer necessary in Python3 (since all strings are unicode by default).